ciao-controller/instance.go

```diff
@@ -193,70 +214,27 @@ func getStorage(c *controller, wl *types.Workload, tenant string, instanceID str
 			return payloads.StorageResource{}, err
 		}
 
-		// don't you need to add support for indicating whether
-		// a block device is bootable.
-		data := types.BlockData{
-			BlockDevice: device,
-			Size:        s.Size,
-			CreateTime:  time.Now(),
-			TenantID:    tenant,
-			Name:        fmt.Sprintf("Storage for instance: %s", instanceID),
-		}
-
-		err = c.ds.AddBlockDevice(data)
-		if err != nil {
-			c.DeleteBlockDevice(device.ID)
-			return payloads.StorageResource{}, err
-		}
+		s.Persistent = true
```

Comment:
That's a very good question.  types.StorageResource's Persistent flag comment is unclear on how the field is used.  If it is set to false, would the storage resource be deleted after completion of an instance of the workload?  If so, given this is in the case for an Image (ie: not a volume), it would seem dangerous to have the image not be persistent and automatically get deleted when a workload depends on it.